Src/Support/Google.Apis.Core/Http/HttpClientFactory.cs

```diff
@@ -23,8 +22,26 @@ namespace Google.Apis.Http
     /// <summary>The default implementation of the HTTP client factory.</summary>
     public class HttpClientFactory : IHttpClientFactory
     {
-        /// <summary>The class logger.</summary>
-        private static readonly ILogger Logger = ApplicationContext.Logger.ForType<HttpClientFactory>();
+        /// <summary>
+        /// Creates a new instance of <see cref="HttpClientFactory"/>.
+        /// </summary>
+        public HttpClientFactory() : this(null)
```

Comment:
I'd be tempted to use a public factory method and a private constructor here (with a public parameterless constructor for backward compatibility), just for future flexibility.